### ISSUE TITLE
Plasma influx from the outer wall boundary due to ExB drifts

### ISSUE THREAD

**Issue by ganjinzero1** [LLNL/UEDGE#61]
**Is your feature request related to a problem? Please describe.**

There would be a strong plasma influx due to ExB on the outer wall near the outer target plate when turning on drifts and the ion Grad-B drift is into the divertor. The net effect of the plasma influx is to increase recycled neutrals in the inner divertor.

This effect of plasma influx caused by ExB seems to be quite substantial in the KSTAR case I have been running. I haven't tested the effect with reverse drifts and not sure if it has a similar effect as the case with forward B. The effect doesn't seem to be big for all the DIII-D cases I have ran, however, it remains to be confirmed.

**Describe the solution you'd like**

One possible solution could be to introduce a flux limiter to limit the plasma influx caused by drifts even though it is somewhat random or artificial.

**Comment by holm10** [comment]
Could you provide a link to an example in the code where the ExB flux is implemented @ganjinzero1 ? Having a few places in the code to start looking at would be a good starting point.

I assume this is part of boundary.m, part of the 'divergence-free' terms we previously discussed, correct? I have implemented a proposed fix for the divergence-free terms on the core boundary here: https://github.com/holm10/UEDGE/tree/fniycbo_netzero I am, however, not sure if the same logic can be applied to the other boundaries, but we could discuss this in more detail at some point.

**Comment by ganjinzero1** [comment]
Yes definitely, I will summarize it with some plots from the KSTAR runs. Probably I can talk about it in one of our upcoming Steering group meetings when we talk about this item:)

**Comment by ganjinzero1** [comment]
As we discussed in our USG meeting, we would like to further investigate this problem, probably together with the core and plate boundary conditions as well.

Just put a note here so that I can come back here to complete this issue once we have some agreements.
